src/libraries/System.Text.Json/gen/Resources/Strings.resx

```diff
@@ -183,6 +183,12 @@
   <data name="JsonConverterAttributeInvalidTypeMessageFormat" xml:space="preserve">
     <value>The 'JsonConverterAttribute' type '{0}' specified on member '{1}' is not a converter type or does not contain an accessible parameterless constructor.</value>
   </data>
+  <data name="DerivedJsonConverterAttributesNotSupportedTitle" xml:space="preserve">
+    <value>Attributes deriving from 'JsonConverterAttribute' are not supported by the source generator.</value>
+  </data>
+  <data name="DerivedJsonConverterAttributesNotSupportedMessageFormat" xml:space="preserve">
+    <value>Attributes deriving from 'JsonConverterAttribute' are not supported by the source generator.</value>
```

Comment:
>>Attributes deriving from 'JsonConverterAttribute' are not supported by the source genera [](http://example.com/codeflow?start=10&length=89)

should this message mention the name of the derived attribute for clarity?

Reply:
I figured it would be redundant since the warning would show up at the attribute, but https://github.com/dotnet/runtime/issues/88424 will change this for types outside the assembly.

Reply:
Fixed.